### PR TITLE
Create buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-…

### PR DESCRIPTION
**Pull request by JJM317**
…rinpoche

### CODE REVIEW

File: _content/articles/buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-rinpoche [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - "Dzongsar Khyentse Rinpoche"
+external_url: "https://www.lionsroar.com/buddhism-nutshell-four-seals-dharma/"
+drive_links:
+  - "https://drive.google.com/file/d/12WiIwAxYAMmD2yS2Osvsp7UHPZO7p2yX/view?usp=sharing"
+formats: [pdf]
+course: view
+tags:
+  - four-seals
+  - tibetan
+  - emptiness
```
Comment (buddhist-uni) [inline _content/articles/buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-rinpoche:13]: emptiness is implied by `view`

File: _content/articles/buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-rinpoche [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - nibbana
+year: 2017
+month: jan
+journal: Lion's Roar
```
Comment (buddhist-uni) [inline _content/articles/buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-rinpoche:17]:
```yaml
journal: lions-roar
```

**Review by buddhist-uni — COMMENTED** [review]
By "... Rinpoche" do you mean to ask whether that should be included in the author's name?

**Comment by JJM317** [comment]
No, but guessing from your question, Rinpoche shouldn't be included? Makes
sense, if so, since it is a title, particularly of respect, rather than a
name. I can remove it. This particular Rinpoche may end up with an author
slug since he is kinda popular in the Tibetan Buddhist orbit.

--------------------

**Comment by buddhist-uni** [comment]
No, I don't mind titles as long as that's how people actually refer to them ("Bhikkhu Bodhi", "H. H. the Dalai Lama", etc) [Somdet Yan](https://buddhistuniversity.net/authors/yan) is maybe a good example where there's the short title in the name but I include all his other names and titles in his author page for #SEO reasons.

No, I was just trying to parse your cryptic comment...

As of now, this is the first work by Khyentse Norbu (right? Not the same guy as Chokyi Lodro, right?) to enter the library, so there's no need to add an author page yet I think 😁

File: _content/articles/buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-rinpoche [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+course: view
+tags:
+  - four-seals
+  - tibetan
+  - nibbana
+year: 2017
+month: jan
+journal: lions-roar
+address: "Halifax, Nova Scotia"
+pages: "7"
```
Comment (buddhist-uni) [inline _content/articles/buddhism-in-a-nutshell-four-seals-of-dharma_dzongsar-khyentse-rinpoche:18]: Page counts not in quotes. (Only ranges go in quotes)

**Comment by JJM317** [comment]
Cool, I'll leave it as is. I think what happened was I was going to ask
this question in the comment box, but thought I would just leave it and see
what you say, and didn't delete the whole thing?

Ya, this looks like Khyentse Norbu's first work in the library so no need.
I was just speculating.

--------------------

**Review by buddhist-uni — APPROVED** [review]
Also, file names must end in `.md`

**Pull request merged**